### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
https://issues.apache.org/jira/browse/IO-806

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/commons-io/actions/runs/5662536320#summary-15342653080

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/commons-io/actions/runs/5662536436#summary-15342654674

---

I can fix the summaries to match the requested style, but at the moment I'm all out of available jobs and I'm waiting for the current jobs to finish.

### CODE REVIEW

File: src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java [hunk 1]
```diff
@@ -410,11 +410,11 @@ public void test_reset_Exception() throws IOException {
...
-        final BufferedInputStream buffis = new BufferedInputStream(new ByteArrayInputStream(input));
-        buffis.read();
-        buffis.mark(5);
-        buffis.skip(5);
-        buffis.reset();
+        final BufferedInputStream bufin = new BufferedInputStream(new ByteArrayInputStream(input));
+        bufin.read();
+        bufin.mark(5);
+        bufin.skip(5);
+        bufin.reset();
```
Comment (jsoref) [inline src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java:417]: This file has a whole bunch of different local variables used for `new BufferedInputStream(...)`. There does not appear to be a remotely good reason for so many different ways to spell the same variable name in a single file, but only this one bothered me enough to change it.

https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L94
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L112
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L143
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L162
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L172
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L216
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L230
-- this is in the same scope as the previous one, but the previous one is effectively dead, so there's no reason not to recycle it as the next two sites do...
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L281
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L302
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L347
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L390
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L428
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L441
https://github.com/apache/commons-io/blob/318645fd97d6c31cfbce6f5a7fa83d5350e87708/src/test/java/org/apache/commons/io/input/UnsynchronizedBufferedInputStreamTest.java#L461

File: src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ private String getName() {
 
     @Test
     public void testDirectoryAndFileContentEquals() throws Exception {
-        // Non-existent files
+        // Nonexistent files
```
Comment (garydgregory) [inline src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java:47]: Let's not change this please, this is not a spelling error, both spellings are acceptable, one is clearer due to the hyphen.

Comment (jsoref) [reply src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java:47]: Sure, but do note that you're already using it in [‎src/main/java/org/apache/commons/io/FileUtils.java](https://github.com/apache/commons-io/blob/06fde31494c279ad940149e1a3d4944040c73c0d/src/main/java/org/apache/commons/io/FileUtils.java#L1091) per https://github.com/search?q=repo%3Aapache%2Fcommons-io%20nonexistent&type=code

Comment (jsoref) [reply src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java]: I've dropped those changes, but I've thrown in some more `existant` -> `existent` changes.

Comment (garydgregory) [reply src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java]: Thank you. I'd rather normalize that word on the hyphenated variant.

Comment (jsoref) [reply src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java]: Sorry, how would you do that? consider https://github.com/apache/commons-io/compare/318645fd97d6c31cfbce6f5a7fa83d5350e87708..810cf30344db9d3e5fffc8ea074d50fae136e558

https://github.com/apache/commons-io/blob/810cf30344db9d3e5fffc8ea074d50fae136e558/src/test/java/org/apache/commons/io/file/PathUtilsTest.java#L102

Comment (garydgregory) [reply src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java]: Err... this is a PR, you edit the files?

Comment (jsoref) [reply src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java]: Then I think this PR has what you want.

Comment (garydgregory) [reply src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java]: It is not. Local vars should be "nonexistentFoo" -> "nonExistentFoo".

**Review by garydgregory — CHANGES_REQUESTED** [review]
See comment.

File: src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java (hunk 2 above)
Comment (jsoref) [reply src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java]: ok, I think this addresses your request

**Review by garydgregory — APPROVED** [review]
LGTM, TY @jsoref

**Pull request merged**